examples/query/canisters/rust/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+#[ic_cdk_macros::query]
+fn simple_query() -> String {
+    format!("This is a query function")
```

Comment:
Use `"This is a query function".to_string()`, format! is a macro with special capabilities, I would say overkill for just returning a string. The convention used in most of the other Rust codebases that I have use .to_string or String::from()